src/storage/concurrency_manager/handle_table.rs

```diff
@@ -0,0 +1,254 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use super::key_handle::{KeyHandle, KeyHandleMutexGuard, KeyHandleRef};
+
+use kvproto::kvrpcpb::LockInfo;
+use parking_lot::Mutex;
+use std::{collections::BTreeMap, ops::Bound, sync::Arc};
+use txn_types::Key;
+
+#[derive(Default)]
+pub struct HandleTable<M>(Arc<M>);
```

Comment:
I don't understand the name - what is the handle? Seems like `LockTable` would be more accurate?